content/en/docs/k8s-operator/automatic.md

```diff
@@ -164,7 +164,7 @@ spec:
     - baggage
   sampler:
     type: parentbased_traceidratio
-    argument: "1"
+    argument: '1'
```

Comment:
@TylerHelmuth - can we drop the quotes here (and at line 237)?

Reply:
I'm not sure if they can be dropped, but single quotes will work.